lyricsheets/effect/default_effect.py

```diff
@@ -37,7 +37,7 @@ def get_char_wrap_tags(
         return ([], [])
 
     return STYLE_FONT_NAME_TO_CHAR_WRAP_TAGS[kChar.style.fontName].get(
-        kChar.text, ([], [])
+        kChar._text, ([], [])
```

Comment:
e.g. these should be able to just use `.text` instead of `._text` which would remove a lot of noise in this PR

Reply:
Updated.